website/docs/reference/resource-configs/persist_docs.md

```diff
@@ -120,11 +120,49 @@ resources as needed.
 
 ## Support
 
-The `persist_docs` config is supported on all core dbt plugins: BigQuery,
-Redshift, Snowflake, and Postgres. Some databases impose limitations on the
-types of descriptions that can be added to database objects. At present, the
-`persist_docs` flag has the following known limitations:
- - Column-level comments are not supported on Snowflake <Term id="view">views</Term>
+The `persist_docs` config is supported on the most widely used dbt adapters:
+- Postgres
+- Redshift
+- Snowflake
+- BigQuery
+- Apache Spark & Databricks
+
+However, some database limit where and how descriptions can be added to database objects. Those database adapters may not support `persist_docs`, or may offer only partial support.
+
+Some known issues and limitations:
+
+<WHCode>
+
+<div warehouse="Databricks">
+
+- Column-level comments require `file_format: delta` (or another "v2 file format")
+- Column-level comments are not supported for models materialized as <Term id="view">views</Term> ([issue](https://github.com/dbt-labs/dbt-spark/issues/372)
+
+</div>
+
+<div warehouse="Snowflake">
+
+<VersionBlock firstVersion="1.2">
+
+- No known issues
+
+</VersionBlock>
```

Comment:
curious, is this version block required? 

from a content perspective, we shouldn't list "no known issues" in a "known issues and limitations" list. if this version block isn't required, suggest we remove it.

Reply:
This is a technical limitation of our current implementation for `VersionBlock` + `WHCode`. There _are_ known issues on Snowflake for earlier versions, but as v1.2, they have been resolved. We cannot nest the "Snowflake" WHCode div within a `VersionBlock`, so it must appear regardless. Would we prefer it be empty for v1.2+, or say "no known issues"?